### DISCUSSION TITLE
Elements inside particles.json file to change based on theme

### DISCUSSION THREAD

**Discussion by m40ma0** [tailwindlabs/tailwindcss#10018] [Help]
![Screenshot (173)](https://user-images.githubusercontent.com/60642680/206395391-ee16a144-72bb-48a6-80b5-1f8d285ec646.png)

As stated, I want the elements inside the particles.json file to change according to the theme. (i.e color, background color) to change according to the theme. Is it possible?

**Comment by wongjn** [comment]
The screenshot provided doesn't give a sufficient overview of how this is mapped to HTML elements or what you mean by "the theme", but you could look at adding Tailwind class names to the JSON file instead of color values, or perhaps use CSS variables.

**Reply by m40ma0** [reply]
Hi, I basically want to make the json file conditional based on the theme chosen by daisyui.

**Reply by wongjn** [reply]
You can't really make a JSON file <q>conditional</q>, but if you mean in your original post whereby you'd like the colors to change according to the Daisy UI theme, you could use the [CSS variables that are generated for each theme color](https://daisyui.com/docs/colors/#-2).
